### ISSUE TITLE
Add permission request prompts

### ISSUE THREAD

**Issue by achpalaman** [minbrowser/min#332]
When using Google Hangouts, how do we grant permissions for it to use the mic and camera? I could not find any popup asking me to give it permissions as is usually the case with other browsers.

**Comment by lawrencenull** [comment]
Agreed this is a pretty big one! Webrtc permissions should be standard.

**Comment by ygkip** [comment]
Is there a work around for this? Some way to accept requests on specific domains or something?

**Comment by PalmerAL** [comment]
There isn't for now, although if you're comfortable with modifying the code you could modify [this file](https://github.com/minbrowser/min/blob/master/main/permissionManager.js) to have something like this:

```
var URL = require("url")

function pagePermissionRequestHandler (webContents, permission, callback) {
  var hostname = URL.parse(webContents.getURL()).hostname
  
  if (hostname === "example.com" && permission === "insert-permission-here") {
        return true
  }

  if (permission === 'notifications' || permission === 'fullscreen') {
    callback(true)
  } else {
    callback(false)
  }
}
```

**Comment by ygkip** [comment]
Thanks for the help. I don't really know what I'm doing though. What am I placing in "insert-permission-here"?

**Comment by PalmerAL** [comment]
Oh, sorry, that's the name of the permission you want to give to the website. The options are:

* media
* geolocation
* midiSysex
* pointerLock
* openExternal

([source](http://electronjs.org/docs/api/session#sessetpermissionrequesthandlerhandler))

**Comment by PalmerAL** [comment]
Fixed in 1.13.2: https://github.com/minbrowser/min/releases/tag/v1.13.2
